core/store/orm/orm.go

```diff
@@ -1333,7 +1341,7 @@ func (orm *ORM) IncrFluxMonitorRoundSubmissions(aggregator common.Address, round
             ?, ?, 0, 1
         ) ON CONFLICT (aggregator, round_id)
         DO UPDATE
-        SET num_submissions = excluded.num_submissions + 1
+        SET num_submissions = flux_monitor_round_stats.num_submissions + 1
```

Comment:
Im surprised about two things:

1. That the original version didn't work
2. That it failed in a way that wasn't caught by tests